### ISSUE TITLE
Convert our `window.alert`s to react-modal

### ISSUE THREAD

**Issue by vanderhoop** [stride-nyc/remote_retro#130]
As a sane user in the year 2017,
I want to interact with alerts and confirmations that don't hurt to look at,
So that I can use remote retro without wondering who developed the UX.

__By Way of Explanation:__ we're currently using `window.alert` to let users know that the action items have been distributed. This was the easiest possible solution at the time, but as we seek to make the application more presentable, we want to leverage react-modal.

- __NOTE:__ this story involves no styling whatsoever. styling will be handled in a follow-up story.

__Acceptance Criteria:__
- [ ] all javascript `alert`s are replaced with modals containing identical copy, along with an "OK" button

__Helpful Links:__

- [react-modal](https://github.com/reactjs/react-modal)

**Comment by vanderhoop** [comment]
__Tasks Remaining:__
- [ ] update end-to-end tests to reflect new user actions
- [ ] kill unused `ModalWrapper` component
- [ ] fix lint

**Comment by vanderhoop** [comment]
moving this to done, as the lone alert we have in the application represents the end of a retro, and updating it at this point provides less value than the rest of the stories on the backlog, some of which are blocked by this story.
